Dockerfile

```diff
@@ -54,7 +47,7 @@ RUN chmod +x /bin/entrypoint-local.sh
 COPY scripts/post-hook.sh /bin/
 
 # We have already built everything, so remove the local repo now
-RUN rm -rf /opt/titan-1.1.0-SNAPSHOT-hadoop2.zip /opt/dynamodb/dynamodb-titan-storage-backend/server/dynamodb-titan100-storage-backend-1.0.0-hadoop1.zip
+#RUN rm -rf /opt/dynamodb/dynamodb-titan-storage-backend/server/janusgraph-0.2.0-hadoop2.zip
```

Comment:
Why is this line commented out?

Reply:
Not sure what we would be gaining by just deleting the jar over here. There is no specific reason apart from that, that i have commented this line

Reply:
In that case we can delete the line. We can always find it in git history, if needed in future.

Reply:
Removed